muikku-core-plugins/src/main/java/fi/otavanopisto/muikku/plugins/workspace/rest/WorkspaceRESTService.java

```diff
@@ -2694,4 +2695,28 @@ public Response deleteWorkspaceFileContent(@PathParam("WORKSPACEID") Long worksp
         .build();
   }
   
+  @GET
+  @Path("/workspaces/{WORKSPACEENTITYID}/activityStatistics")
+  @RESTPermit(handling = Handling.INLINE)
```

Comment:
Inline permissions but not checked. Should probably check that logged user can check their own statistics or if the logged user has some specific permission (workspace teacher? some high-up environment roles?).

Reply:
Permission check added

Reply:
Where?....

Reply:
Oh yes sorry, this didn't display all the lines... I think you should poke viljakainen to resolve this.